R/bit64.R

```diff
@@ -0,0 +1,126 @@
+#' @export
```

Comment:
Actually this should be in `type-integer64.R` — can you have a look at the other `type-*.R` files and match their sections?

I think you might also want to make a `new_integer64()` constructor (or maybe `new_int64()`)